Project.toml

```diff
@@ -7,9 +7,11 @@ version = "0.2.1"
 BioStructures = "de9282ab-8554-53be-b2d6-f6c222edabfc"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
```

Comment:
Add compat bounds to this file too, e.g. 0.4 for NearestNeighbors and 0.10 for Distances.

Reply:
Done.